Command/DoctrineCommandHelper.php

```diff
@@ -56,11 +56,14 @@ public static function setApplicationConnection(Application $application, $sessi
      * @param Application $application
      * @param string      $sessionName
      */
-    public static function setApplicationPHPCRSession(Application $application, $sessionName)
+    public static function setApplicationPHPCRSession(Application $application, $sessionName, $admin = false)
     {
-        /** @var $registry ManagerRegistry */
-        $registry = $application->getKernel()->getContainer()->get('doctrine_phpcr');
-        $session = $registry->getConnection($sessionName);
```

Comment:
we cannot use the registry here anymore, since the admin sessions are namespace differently .. we could of course just use a naming convention for the admin sessions following the normal "scheme" but this could in theory mean that we have conflicts between user defined sessions and automatically defined admin sessions.